models/v1/gardens/gardens_models.go

```diff
@@ -0,0 +1,8 @@
+package gardens_models
+
+type Rules struct {
```

Comment:
@yukari-n-erb You can change it in another PR but it's probably supposed to be `Gardens`, not Rules.